modular_skyrat/modules/holidays/flora.dm

```diff
@@ -0,0 +1,11 @@
+/obj/structure/flora/tree/pine/xmas/presents/safe
+	icon_state = "pinepresents"
+	desc = "A wondrous decorated Christmas tree. It has presents!"
+	gift_type = /obj/item/a_gift //only give safe gifts from the tree
+	unlimited = FALSE
+
+/obj/structure/flora/tree/pine/xmas/presents/safe/unlimited
+	icon_state = "pinepresents"
+	desc = "A wondrous decorated Christmas tree. It has presents!"
+	gift_type = /obj/item/a_gift //only give safe gifts from the tree
+	unlimited = TRUE
```

Comment:
File isn't ticked, which is causing lints to fail.

Reply:
Oh I reverted changes to tgstation.dme, so yeah. Derp.